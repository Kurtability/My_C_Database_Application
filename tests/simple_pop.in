pop a
pop b
set a 1 2 3
pop a
get a
pop a 1
get a
pop a 1
get a
pop a 1
get a
bye
