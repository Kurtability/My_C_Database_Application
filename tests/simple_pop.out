> pop a 
no such key

> pop b
no such key

> set a 1 2 3 
ok

> pop a 
1

> get a 
[2 3]

> pop a 1 
2

> get a 
[3]

> pop a 1
3

> get a 
[]

> pop a 1
nil

> get a 
[]

> bye
bye
